common/scripted_triggers/wc_scripted_triggers.txt

```diff
@@ -208,8 +208,8 @@ can_switch_religion_character_trigger = {
 # CAN'T ADOPT SECRET RELIGION IF IT'S NOT TRUE
 can_adopt_secret_religion_character_trigger = {
 	holy_order = no
-	controls_religion = no
-	is_theocracy = no
+	#controls_religion = no
+	#is_theocracy = no
```

Comment:
Theocracies lose titles when they change religion so it's better for their own sake and sake of their secret religions to not publicly adopt it.